qiskit_sphinx_theme/translations.py

```diff
@@ -12,50 +12,28 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-# THIS EXTENSION IS BASED ON versionutils.py IN qiskit-metapackage/docs
-
 from functools import partial
 
-from sphinx.util import logging
-
-
-logger = logging.getLogger(__name__)
-
-translations_list = [
-    ('en', 'English'),
-    ('bn_BN', 'Bengali'),
-    ('fr_FR', 'French'),
-    ('de_DE', 'German'),
-    ('ja_JP', 'Japanese'),
-    ('ko_KR', 'Korean'),
-    ('pt_UN', 'Portuguese'),
-    ('es_UN', 'Spanish'),
-    ('ta_IN', 'Tamil'),
-]
-
 default_language = 'en'
 
 
 def setup(app):
     app.connect('config-inited', _extend_html_context)
-    app.add_config_value('content_prefix', '', '')
-    app.add_config_value('translations', True, 'html')
-    return dict(parallel_read_safe=True)
+    app.add_config_value("content_prefix", default="", rebuild="", types=[str])
+    app.add_config_value("translations_list", default=[], rebuild="html", types=[list])
 
 
 def _extend_html_context(app, config):
     context = config.html_context
-    context['translations'] = config.translations
-    context['translations_list'] = translations_list
-    context['current_translation'] = _get_current_translation(config) or config.language
```

Comment:
Do you know if `context['current_translation']` is now used else where? 

It was added, IIRC, for the page to remember the language the user was on, when they go from one page to another.

Reply:
I couldn't find anywhere it's used. `html_context` exists so that HTML templates can reference values that are inserted by Python. 

Our templates are using `language_label` instead. If you look at the Python implementation, `language_label` and `current_translation` are computed almost the same way.

Reply:
I think it's used in qiskit: https://github.com/Qiskit/qiskit-metapackage/blob/a6a6eb885939ff26e59b77d1e22c8fca9b55ed9d/docs/versionutils.py#LL55C89-L55C89

Reply:
It is indeed _set_ by Qiskit and all the ecosystem projects. But it is never _consumed_ by anything. We're setting a value for no benefit, which is confusing.

Reply:
ah ok, let's remove it for now but make a mental note if we find something breaks for someones project we might need to add it back

Reply:
Sounds good!